tests/integration/servicemesh/smmr_test.go

```diff
@@ -166,16 +211,16 @@ spec:
   hosts:
   - "%s.maistra.io"
   gateways:
-  - %s/common-gateway
+  - %s/%s
   http:
   - route:
     - destination:
         host: localhost
         port:
           number: 8080
-`, virtualServiceName, virtualServiceName, gatewayNs)
+`, virtualServiceName, virtualServiceName, gatewayNs, gatewayName)
```

Comment:
Instead of duplicating arguments here you could use positional and refer to virtual service name using `%[1]s`, so that would become sth like

```
metadata:
  name: %[1]s
spec:
  hosts:
  - "%[1]s.maistra.io"
  gateways:
  - %[2]s/%[3]s
```